rplugin/python3/deoplete/sources/ternjs.py

```diff
@@ -314,7 +303,8 @@ def completation(self, pos):
             for rec in data["completions"]:
                 completions.append({
                     "word": rec["name"],
-                    "menu": self.completion_icon(rec.get("type")),
+                    "kind": self.completion_icon(rec.get("type")),
+                    "abbr": rec.get("type").replace('fn', rec["name"], 1),
```

Comment:
I don't know what benefit we will have with re, this is so simple to complicate it with that or what are your thoughts?